### PR TITLE
AISDK-30: Add custom vocabulary support

### PR DESCRIPTION
**Pull request by kbridbur**

### CODE REVIEW

File: src/rev_ai/apiclient.py [hunk 1]
```diff
@@ -84,14 +89,17 @@ def submit_job_local_file(
...
+            skip_diarization=False,
+            custom_vocabularies=None):
         """Submit a local file for transcription.
         Note that the content type is inferred if not provided.
 
         :param filename: path to a local file on disk
         :param metadata: info to associate with the transcription job
         :param callback_url: callback url to invoke on job completion as a webhook
         :param skip_diarization: should rev.ai skip diaization when transcribing this file
+        :param custom_vocabularies: list of dictionaries containing a mapping from 'phrases'
```
Comment (amikofalvy) [inline src/rev_ai/apiclient.py]: Take the wording from the documentation.yaml file

File: src/rev_ai/apiclient.py [hunk 2]
```diff
@@ -54,14 +54,20 @@ def submit_job_url(
...
         """Submit media given a URL for transcription.
         The audio data is downloaded from the URL.
 
         :param media_url: web location of the media file
         :param metadata: info to associate with the transcription job
         :param callback_url: callback url to invoke on job completion as a webhook
         :param skip_diarization: should rev.ai skip diaization when transcribing this file
+        :param custom_vocabularies: a collection of phrase dictionaries. Including custom 
+            vocabulary will inform and bias the speech recognition to find those phrases. 
+            Each dictionary should consist of a key "phrases" which maps to a list of strings,
```
Comment (kelly-liu) [inline src/rev_ai/apiclient.py]: *Each dictionary should have the key "phrases"....

File: tests/test_job.py [hunk 3]
```diff
@@ -135,7 +136,8 @@ def test_submit_job_url_with_success(self, mock_client, make_mock_response):
             json={
                 'media_url': MEDIA_URL,
                 'callback_url': CALLBACK_URL,
-                'metadata': METADATA
+                'metadata': METADATA,
```
Comment (kelly-liu) [inline tests/test_job.py]: i think the skip_diarization param should also be included in this json?

**Comment by kbridbur** [comment]
Sorry @kelly-liu, missed your comments. I created a new PR to address them https://github.com/revdotcom/revai-python-sdk/pull/23

**Pull request merged**